src/java/fr/paris/lutece/portal/web/user/AuthenticationFilter.java

```diff
@@ -157,10 +168,62 @@ public void doFilter( ServletRequest request, ServletResponse response, FilterCh
                 }
             }
         }
+        else if(isAdminLogoutUrl(req))
+        {
+        	addLogoutPageSecurityHeaders(req, resp);	
+        }
 
         chain.doFilter( request, response );
     }
 
+    /**
+     * Adds http security headers to BO admin authenticated pages.
+     * 
+     * @param request
+     *            the http request
+     * @param response
+     *            the http response
+     */
+    private void addAdminAuthenticatedPagesHeaders( HttpServletRequest request, HttpServletResponse response )
+    {
+    	SecurityHeaderService securityHeaderService = SpringContextService.getBean( "securityHeaderService" );
+    	for( SecurityHeader securityHeader : securityHeaderService.findActive( SecurityHeaderType.PAGE.getCode( ), SecurityHeaderPageCategory.AUTHENTICATED_ADMIN_BACK_OFFICE.getCode( ) ) )
+    	{
+    		response.setHeader( securityHeader.getName( ), securityHeader.getValue( ) );
+    		_logger.debug( "Security header added to admin authenticated BO page {} - name : {}, value : {} ", request.getServletPath( ), securityHeader.getName( ), securityHeader.getValue( ) );
+    	}
+    }
+    
+    /**
+     * Adds http security headers to BO admin logout page.
+     * 
+     * @param request
+     *            the http request
+     * @param response
+     *            the http response
+     */
+    private void addLogoutPageSecurityHeaders( HttpServletRequest request, HttpServletResponse response )
+    {
+    	SecurityHeaderService securityHeaderService = SpringContextService.getBean( "securityHeaderService" );
```

Comment:
Please use a static constant.

Reply:
Taken into account in commit 93a56b8139c008a7cc3a87cce5a8ccbffb4c9734